appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/MockForm.java

```diff
@@ -570,65 +570,6 @@ private void resizePanel(int newWidth, int newHeight){
     }
   }
 
-  /*
-   * Changes the preview of MockForm based on Android Holo, Android Material and iOS styles
-   */
-  private void changePreview() {
```

Comment:
Please review this change. It might be the source of the internal error I tripped on when trying to use iOS preview style in the designer.